.travis.yml

```diff
@@ -1,25 +1,28 @@
 language: c
 
+env:
+- CFLAGS="-Wall -Wextra -Wunused -Wimplicit -Wreturn-type -Wdeclaration-after-statement -Wno-unknown-pragmas -Wmissing-prototypes -Wmissing-declarations -Wparentheses -Wswitch -Wtrigraphs -Wcast-qual -Wcast-align -Wwrite-strings -Wold-style-definition -Wpointer-arith"
```

Comment:
We also need to pass such flags to the buildbot.

Reply:
Then you need to adjust the buildbot configuration.  Is there a separate configuration file for it?  We could also add a bash script which sets `CFLAGS` and executes the `script:` section from the Travis config.

Reply:
It's currently under https://github.com/TeXLive-M/texlive-buildbot (temporary buildbot branch).